app-foundation/3-networks/development/README.md

```diff
@@ -0,0 +1,40 @@
+# 3-networks/development
+
+This is a terraform wrapper for [CFT 3-Networks/envs/development](https://github.com/terraform-google-modules/terraform-example-foundation/tree/master/3-networks/envs/development) and can be used to set up a base shared VPC with default NAT, baseline firewall rules, Cloud Armor, private service networking, and an external IP address for the Bank of Anthos example application in the development environment.
+
+## Prerequisites
+
+1. 0-bootstrap executed successfully.
+1. 1-org executed successfully.
+1. 2-environments/envs/development executed successfully.
+1. 3-networks/envs/shared executed successfully.
+1. The two local variables defined in this example are changed in [3-Networks/envs/development/main.tf](https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/3-networks/envs/non-production/main.tf).
```

Comment:
Add instructions before this to be within the correct 3-networks env folder. Lets also add the code block for locals as discussed offline